ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity-tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-workflow.yaml",
+	"path": "cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity-tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml",
```

Comment:
I suggest to create a new folder - `cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/`, so that, in the future, we can easily switch over.